deploy/effective-dart-rules/pubspec.yaml

```diff
@@ -1,9 +1,13 @@
 name: effective_dart_rules
-version: 0.0.2
+version: 0.0.3
 description: Builds a file with all the rules in the different sections of Effective Dart.
 environment:
-  sdk: '>=2.10.0 <3.0.0'
+  sdk: '>=2.13.0 <3.0.0'
```

Comment:
I'm curious if this code uses Dart 2.13 specific code, or if 2.12 would have been sufficient.